examples/k8s-health-check/proxy_with_http_health_check.yaml

```diff
@@ -49,90 +49,104 @@ spec:
       - name: cloud-sql-proxy
         # It is recommended to use the latest version of the Cloud SQL Auth Proxy
         # Make sure to update on a regular schedule!
-        image: gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.1.0
+        image: gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.6.0
+        imagePullPolicy: IfNotPresent
+
         args:
-          # If connecting from a VPC-native GKE cluster, you can use the
-          # following flag to have the proxy connect over private IP
-          # - "--private-ip"
+        # Replace <INSTANCE_CONNECTION_NAME> with the instance connection
+        # name in the format: "project_name:region:instance_name"
+        # Replace <DB_PORT> with the port that the proxy should open
+        # to listen for database connections from the application
+        - <INSTANCE_CONNECTION_NAME>
```

Comment:
FYI This could be an environment variable too. `CSQL_PROXY_INSTANCE_CONNECTION_NAME=...`

Reply:
Not sure if we want to do that, though.

Reply:
I avoided that in the operator's implementation because  when configuring multiple instances, the envvar was somewhat more confusing than the args.

Reply:
this would be highly appreciated to have. We can't move from 1.x to 2.x because of this missing. I even opened a bugticket through the support: https://issuetracker.google.com/issues/273982124

Reply:
@w32-blaster would you mind opening an issue so we can track that here?